mantidproject/mantid
# ISIS Reflectometry GUI: Fixed bug where unexpected error was thrown when processing with output notebook option

As described in the related issue, attempting to process runs with the `Output Notebook` option selected will throw an unexpected error. This PR fixes this problem so an IPython notebook is generated normally without error.

**To test:**
- Access the interface from Interfaces->Reflectometry->ISIS Reflectometry
- Obtain some sample data (e.g. import INTER_NR_test2.tbl from ISIS sample datasets).
- Check the Output Notebook option at the bottom of the Runs tab.
- Process some data - a window for saving the notebook should appear. Enter some name for the file.
- It should save the notebook normally without throwing an error.



Fixes #19245.



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
